R/vctrs.R

```diff
@@ -0,0 +1,370 @@
+#' vctrs methods for lubridate
+#'
+#' @keywords internal
+#' @name lubridate-vctrs
+NULL
+
+# ------------------------------------------------------------------------------
+# Constructors
+
+# TODO: Find a way to generate these once at load time for efficiency. S4
```

Comment:
What about with `delayedAssign()`? (we know it's not triggered by package building)